packages/core/admin/admin/src/pages/ProfilePage/index.js

```diff
@@ -280,155 +304,162 @@ const ProfilePage = () => {
-                                      : {
-                                          id: 'Auth.form.password.hide-password',
-                                          defaultMessage: 'Hide password',
-                                        }
-                                  )}
-                                >
-                                  {passwordShown ? <Eye /> : <EyeStriked />}
-                                </FieldActionWrapper>
-                              }
-                            />
-                          </GridItem>
-                          <GridItem s={12} col={6}>
-                            <PasswordInput
-                              error={
-                                errors.confirmPassword
-                                  ? formatMessage({
-                                      id: errors.confirmPassword,
-                                      defaultMessage: errors.confirmPassword,
-                                    })
-                                  : ''
-                              }
-                              onChange={handleChange}
-                              value={values.confirmPassword || ''}
-                              label={formatMessage({
-                                id: 'Auth.form.confirmPassword.label',
-                                defaultMessage: 'Password confirmation',
-                              })}
-                              name="confirmPassword"
-                              type={passwordConfirmShown ? 'text' : 'password'}
-                              autoComplete="new-password"
-                              endAction={
-                                <FieldActionWrapper
-                                  onClick={(e) => {
-                                    e.stopPropagation();
-                                    setPasswordConfirmShown((prev) => !prev);
-                                  }}
-                                  label={formatMessage(
-                                    passwordConfirmShown
-                                      ? {
-                                          id: 'Auth.form.password.show-password',
-                                          defaultMessage: 'Show password',
-                                        }
-                                      : {
-                                          id: 'Auth.form.password.hide-password',
-                                          defaultMessage: 'Hide password',
-                                        }
-                                  )}
-                                >
-                                  {passwordConfirmShown ? <Eye /> : <EyeStriked />}
-                                </FieldActionWrapper>
-                              }
-                            />
-                          </GridItem>
-                        </Grid>
-                      </Flex>
-                    </Box>
+                    {
```

Comment:
Maybe having this condition here is a good opportunity to extract this to a new component and reduce the file size? what do you think? 🤔

Reply:
done